javascript/javascript1/week3/Js1week3.js

```diff
@@ -0,0 +1,189 @@
+//Item array removal
+
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+];
+const nameToRemove = "Ahmad";
+const indexNameToRemove = names.indexOf("Ahmad");
+names.splice(indexNameToRemove, 1);
+
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala'] */
```

Comment:
Great work! 💫
Maybe the variable naming of const indexNameToRemove could simply be const indexToRemove